### PR TITLE
finalized the Leaderboard list app, adding the styles needed to make it look great

### PR DESCRIPTION
**Pull request by Mhdez221993**
improved the look and feel of the application, adding the style
kept the general layout of the wireframe
used plain CSS and Boostrap 5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service.
 
 
-![screenshot](https://raw.githubusercontent.com/microverseinc/curriculum-javascript/main/leaderboard/images/leaderboard_wireframe.png?token=AQE6HSPX3NJHSIM2YAMRRKLBBE7V6)
+![screenshot](leaderboard.png)
 
 ## Built With
```
Comment (sinansevgi) [inline README.md:8]: - [x] You should also add instructions for using consumed API service. (where and how to generate any necessary keys, credentials, how to set, store, and generate them) 

Consider someone who is not from the microverse may want to check your project.
If I want to use your app for myself, currently there is no way to know how to configure (API) it.

**Review by sinansevgi — CHANGES_REQUESTED** [review]
# Changes Required :recycle: 
------

Hi @Mhdez221993 , 

Good job so far!

Styling looks really good :+1:  
But you should improve your readme a little bit to make it more professional. Please check my inline comment about this.

 ## Required Changes ♻️
_Please check the inline comments for my suggestions_ :arrow_down:

Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi

------

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Davidosky007** [comment]
# Hi @Mhdez221993 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md (hunk 1 above)
Comment (Davidosky007) [reply README.md:8]: done 👍

**Pull request merged**